prompts/template_xml/context.txt

```diff
@@ -1,8 +1,4 @@
-The function is defined in
-<include_file>
-{CONTEXT_HEADER}
-</include_file>
-
+You must insert this code before the function being tested:
```

Comment:
1. How about other context info, such as [function implementation](https://github.com/google/oss-fuzz-gen/blob/main/data_prep/introspector.py#L135) and [cross references](INTROSPECTOR_XREF)?

2. LLM does not have to include all context info, such as the two above.

Reply:
added